src/linker/pipeline_schema_constants.py

```diff
@@ -0,0 +1,49 @@
+from linker.utilities.data_utils import validate_dummy_input, validate_dummy_output
+
+ALLOWED_SCHEMA_PARAMS = {
+    "pvs_like_case_study": {
+        "validate_input": lambda *_: None,
+        "steps": {
+            "pvs_like_case_study": {
+                "validate_output": lambda *_: None,
+                "inputs": {
+                    "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS": {
+                        "container_dir_name": "/input_data/main_input",
+                        "input_filenames": ["file1"],
+                        "prev_output": False,
+                    }
+                },
+            }
+        },
+    },
+    "development": {
+        "validate_input": validate_dummy_input,
+        "steps": {
+            "step_1": {
+                "validate_output": validate_dummy_output,
+                "inputs": {
+                    "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS": {
+                        "container_dir_name": "/input_data/main_input",
+                        "input_filenames": ["file1"],
```

Comment:
This looks better! I am curious how this will extend to the multiple-files-in-one-slot concept, but we can deal with that when we get there.

Reply:
I'm thinking (hoping?) that it will be pretty straightforward to generalize this to lists of files both in terms of the input slots (we can add any number of paths to a slot with the `add_bindings` method) and in terms of the input data yaml (just have keys take in lists of files instead of single files). That is, if we think that's the appropriate way to extend such fuctionality.